src/registries/npm-registry.js

```diff
@@ -130,7 +130,7 @@ export default class NpmRegistry extends Registry {
     const possibles = [
       [false, path.join(this.cwd, filename)],
       [true, this.config.userconfig || path.join(userHome, filename)],
-      [false, path.join(getGlobalPrefix(), filename)],
+      [false, path.join(getGlobalPrefix(), 'etc', filename.slice(1))],
```

Comment:
Please add a comment about the `.slice(1)` part above this line. Or even better, add example paths as comments:
```
./.npmrc
~/.npmrc
$PREFIX/etc/npmrc
```